prioritize.lua

```diff
@@ -200,6 +200,33 @@ local function boost(job_matchers, opts)
     end
 end
 
+local function boost_oldest(job_matchers, opts)
+    local quiet = opts.quiet
+    local count = opts.oldest_count
+    local job = df.global.world.jobs.list
+    while ( job.next ) do
+        job = job.next
+        if (job.item and not job.item.flags.do_now and job.item.order_id == -1) then
```

Comment:
I see this excludes manager orders. should it also exclude repeat jobs? what about jobs that already have a worker attached and are in progress?

Reply:
it probably should preclude repeat jobs, i don't use those in my forts except very rarely so it did not come up for me

the others are also good ideas